setup/README.adoc

```diff
@@ -4,7 +4,7 @@ This document describes how to use the setup tool to set up a Dev Sandbox enviro
 
 == Prereqs
 
-. Provision the *latest available* GA version of OCP on AWS with sufficient resources: 3 `m5.4xlarge` master nodes and 3 `m5.2xlarge` worker nodes.
+. Provision the *latest available* GA version of *OCP 4.10.x* on AWS with sufficient resources: 3 `m5.4xlarge` master nodes and 3 `m5.2xlarge` worker nodes.
```

Comment:
We had to resize our production clusters to m5.8large due to actual API server memory consumption. So I'm thinking maybe we should change the recommended size for the performance tests too...
WDYT?

Reply:
Makes sense

Reply:
Done in 74c3babc382cff841adb2468ac9bdf0f3be52808